Cargo.toml

```diff
@@ -31,9 +31,9 @@ similar_names = "allow"
 too_many_lines = "allow"
 
 [workspace.dependencies]
-ndc-models = { git = "https://github.com/hasura/ndc-spec.git", tag = "v0.1.6" }
-ndc-sdk = { git = "https://github.com/hasura/ndc-sdk-rs.git", tag = "v0.4.0" }
-ndc-test = { git = "https://github.com/hasura/ndc-spec.git", tag = "v0.1.6" }
+ndc-models = { git = "https://github.com/hasura/ndc-spec.git", tag = "v0.2.0-rc.2" }
+ndc-sdk = { git = "https://github.com/hasura/ndc-sdk-rs.git", rev = "643b96b" }
```

Comment:
Can we bump `ndc-sdk` separately before this PR goes in?

Reply:
That's correct, the only reason we bump it up is to bring in the new models. We want to make sure the `ndc-models` we import directly and the ones from the SDK are the same version.